### PR TITLE
build: fail the build

### PR DESCRIPTION
**Pull request by kanej**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Lint and Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
```
Comment (kanej-test-1) [inline .github/workflows/test.yml:8]: An inline comment about jobs

Comment (kanej) [reply .github/workflows/test.yml:8]: Another comment

**Comment by kanej** [comment]
A general comment.

File: .github/workflows/test.yml (hunk 1 above)
Comment (kanej-test-1) [reply .github/workflows/test.yml:8]: Okay sounds good

Comment (kanej-test-1) [reply .github/workflows/test.yml:8]: A further comment

**Comment by kanej** [comment]
Once more with feeling

**Comment by kanej** [comment]
Here is a comment, from mailscript!

**Pull request closed without merging**